protocol/messenger.go

```diff
@@ -77,6 +78,8 @@ const (
 
 	publicChat  chatContext = "public-chat"
 	privateChat chatContext = "private-chat"
+
+	isMobileApp = runtime.GOOS == "android" || runtime.GOOS == "ios"
```

Comment:
not sure if this will work, have you tested? 🙂

Reply:
No, I haven't tested it, but don't see the reason why it should not work? (hope this is a temporary here, till mobile app aligns with the desktop app 🙂)
If you have any better idea to achieve the same let me know and I am ready to change it?

Reply:
How about having this flag named after its final purpose (e.g. enableWalletAccountSync) and be set from the API by the mobile?